### DISCUSSION TITLE
我想帮忙升级react-sticky-box到1.x版本

### DISCUSSION THREAD

**Discussion by bobfw** [alibaba/formily#3100] [RFC]
react-sticky-box 1.0 版本中删除了onChangeMode方法，这个方法在formily有被声明但是无地方使用，文档也没有暴露，可以直接删除吗

**Comment by janryWang** [comment]
升级的收益是？

**Reply by bobfw** [reply]
> 升级的收益是？
解决对等依赖问题，pnpm 每次安装都报警告，不优雅，
react-sticky-box 0.9.x 
  "peerDependencies": {
    "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
    "prop-types": "^15.6.2"
  }

**Reply by janryWang** [reply]
你可以先升级看看，跑下文档，看看文档例子有没有影响
